### ISSUE TITLE
Travis Build never fails, even if tests do

### ISSUE THREAD

**Issue by harman28** [kenpratt/wikipedia-client#56]
Our Travis build never fails! Even when the tests do!
It seems `bundle exec rake` always exits with code 0, no matter what.

You can see this behaviour [here](https://travis-ci.org/kenpratt/wikipedia-client/builds/235054310), where a build passes, even though one test has obviously failed.
> The command "bundle exec rake" exited with 0.

I also tried it on local.

```sh
wikipedia-client (development) > bundle exec rake spec
...
<lots of failure>
...
Finished in 0.00658 seconds
21 examples, 1 failure, 20 pending

Failed examples:

rspec ./spec/lib/client_spec.rb:65 # Wikipedia::Client.find page with one section (mocked) should have the correct sanitized intro
wikipedia-client (development) > echo $?
0 #=> Exit code is zero, even thought tests failed
```

Will investigate this further. It might be an `rspec` version issue, as ours is quite old, and I'm seeing mentions of such errors here:
 - https://github.com/colszowka/simplecov/issues/281
 - https://github.com/rspec/rspec-rails/issues/1016
 - https://github.com/allure-framework/allure1/issues/884

**Comment by harman28** [comment]
Fixed in #57

**Comment by pietromenna** [comment]
This issue is solved by #57
